OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -108,7 +117,8 @@ public static Ruleset LoadDefaults(ModData modData)
 			Action f = () =>
 			{
 				var actors = MergeOrDefault("Manifest,Rules", fs, m.Rules, null, null,
-					k => new ActorInfo(modData.ObjectCreator, k.Key.ToLowerInvariant(), k.Value));
+					k => new ActorInfo(modData.ObjectCreator, k.Key.ToLowerInvariant(), k.Value),
+					filterNode: n => n.Key.StartsWith(ActorInfo.AbstractActorPrefix, StringComparison.Ordinal));
```

Comment:
I went with explicit param identifier for readability, as the actor case is the only one opting into the filter pattern, and I didn't want to spam the others with boilerplate.